tools/build_api.py

```diff
@@ -457,6 +457,7 @@ def build_library(src_paths, build_path, target, toolchain_name,
         toolchain.copy_files(resources.headers, build_path, resources=resources)
         toolchain.copy_files(resources.objects, build_path, resources=resources)
         toolchain.copy_files(resources.libraries, build_path, resources=resources)
+        toolchain.copy_files(resources.json_files, build_path, resources=resources)
```

Comment:
I believe the paths for `mbed_lib.json` files should be preserved, though I will double check this!